packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -20,6 +20,8 @@ const ZoeSeatIKit = harden({
     fail: M.call(M.any()).returns(),
     resolveExitAndResult: M.call(M.promise(), M.remotable('exitObj')).returns(),
     getExitSubscriber: M.call().returns(SubscriberShape),
+    // The return promise is empty, but doExit relies on resolution as a signal
```

Comment:
Technically "settlement" or "fulfillment" rather than "resolution".

The leaves of taxonomy are that an unresolved promise can be
   * "forwarded" to another promise, which may be unresolved. Its fate then mirrors the fate of the promise it was forwarded to.
   * "fulfilled" resolved to a non-promise non-thenable
   * "rejected" with a reason.

The non-leaves of the taxonomy, a promise is
   * "resolved" if it is forwarded, fulfilled, or rejected
   * "settled" if it is fulfilled or rejected

A `then` or `E.when` can only react to the promise being settled. From the promise alone, one should not be able to tell whether the promise has been forwarded.

Reply:
Thanks. If I once knew, I had forgotten the distinction between "resolved" and "settled".  i.e. I had forgotten about the "forwarded" case.

Reply:
Looks done. Resolving.